graylog2-server/src/main/java/org/graylog2/bootstrap/ServerBootstrap.java

```diff
@@ -157,7 +159,7 @@ protected void savePidFile(final String pidFile) {
                 throw new Exception("Could not determine PID.");
             }
 
-            Files.write(pidFilePath, pid.getBytes(StandardCharsets.UTF_8));
+            Files.write(pidFilePath, pid.getBytes(StandardCharsets.UTF_8), StandardOpenOption.WRITE, StandardOpenOption.CREATE_NEW, LinkOption.NOFOLLOW_LINKS);
```

Comment:
Wouldn't it make sense to use `CREATE` and `TRUNCATE_EXISTING` instead of `CREATE_NEW` in this place? If the file already exists and `CREATE_NEW` is being used, the call would fail instead of overwriting the specified file.